core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -547,7 +565,7 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     assertEquals(2, parts.size)
   }
 
-  @Test
+  @Test // TODO: doesn't pass for kraft and kraft+kip848
```

Comment:
This test doesn't pass for kraft and kraft+kip848. If we sleep for 5 sec between L572 and L573, the test passes. So I guess something is setting up or running in the background and the operation is not ready when L573 is called.

I didn't migrate this because it's not related to the group coordinator. Do we want to look into it further and migrate this as well?

Reply:
`consumer.partitionsFor("non-exist-topic")` is currently not supported in kraft